http/src/main/resources/reference.conf

```diff
@@ -645,6 +645,12 @@ gke {
     autoscalingEnabled = false
     maxNodepoolsPerDefaultNode = 16
   }
+  defaultAutoScalingNodepool {
```

Comment:
Could you leave a comment here around the need o change the machine type and the number of nodes? Like a little summary of what you shared on slack
```regarding pause/resume GKE apps, I was able to get autopause to behave for the first app in the cluster…but I had to do a few tweaks in the default nodepool
move nginx to default nodpeool (making default nodepool to n1-standard-4 suffice nginx’s needs)
make sure metrics-server is moved to default nodepool (for this to work, I had to make the default nodepool having 2 nodes instead of 1)
so in short, this will increase cost for running an app on a GKE cluster…
default nodepool used to be 1 node of  n1-standard-1 ($34.6748175/month)..and now it’ll be 2 nodes of n1-standard-4 ($138.69927*2/month)```